### PR TITLE
Fix cronos

### PR DESCRIPTION
**Pull request by defisaur**
https://github.com/trustwallet/assets/issues/25572

### CODE REVIEW

File: validation/info/fields_validators.go [hunk 1]
```diff
@@ -51,11 +51,16 @@ func ValidateAssetRequiredKeys(a AssetModel) error {
...
 func ValidateAssetType(assetType string, chain coin.Coin) error {
-	chainFromType, err := types.GetChainFromAssetType(assetType)
+	chainFromTypeRaw, err := types.GetChainFromAssetType(assetType)
 	if err != nil {
 		return fmt.Errorf("failed to get chain from asset type: %w", err)
 	}
 
+	chainFromType := chainFromTypeRaw
+	if chainFromTypeRaw.ID == coin.CRONOS {
+		chainFromType = coin.Cryptoorg()
```
Comment (iCell) [inline validation/info/fields_validators.go:61]: can we update this information on `assets` repo?

Comment (defisaur) [reply validation/info/fields_validators.go:61]: unfortunately we can't without copypasting whole validate func from here

Comment (iCell) [reply validation/info/fields_validators.go:61]: wait, I haven't approved this PR.

I saw this PR https://github.com/trustwallet/assets/pull/25575/files is fixing the wrong directory, so why do we still need this change?

Comment (defisaur) [reply validation/info/fields_validators.go:61]: In `check` CI action we had 2 type of errors:
1. Invalid explorer url. It was fixed by https://github.com/trustwallet/assets/pull/25575/files
2. When trying to get corresponding chain to CRC20 token, it returns Cronos which causes other type of validation error. It was fixed by this line

Comment (iCell) [reply validation/info/fields_validators.go:61]: Got it, thank you.
May I ask you to add comments above your changes?

Comment (defisaur) [reply validation/info/fields_validators.go:61]: Sure, added https://github.com/trustwallet/assets-go-libs/pull/103

**Pull request merged**